recipe/meta.yaml

```diff
@@ -50,7 +50,9 @@ requirements:
     - r-magrittr
     - r-s2 >=1.1.0
     - r-units >=0.7_0
-    - libgdal >=3.6.0
+    - libgdal >=3.6.2
+    - geos >=3.8.0
+    - proj >=9.3.1
```

Comment:
Aren't these pulled in transitively by `libgdal`?

(Ditto for `requirements.run`)

Reply:
They are, but I believe they should be here as well since they are specifically dependencies of sf. Since they both have `run_exports` they probably shouldn't be listed in runtime requirements, though.